tools/Makefile

```diff
@@ -78,13 +77,13 @@ else ifeq ($(OS),Windows_NT)
   # This new OS path is confirmed working on Windows 10 / Cygwin64.
   openfst_add_CXXFLAGS = -g -O2 -Wa,-mbig-obj
 else
-  openfst_add_CXXFLAGS = -g -O2
+  openfst_add_CXXFLAGS = -g -O3
 endif
 
 openfst-$(OPENFST_VERSION)/Makefile: openfst-$(OPENFST_VERSION)
 	cd openfst-$(OPENFST_VERSION)/ && \
 	./configure --prefix=`pwd` $(OPENFST_CONFIGURE) CXX="$(CXX)" \
-		CXXFLAGS="$(CXXFLAGS) $(openfst_add_CXXFLAGS)" \
+		CXXFLAGS="$(openfst_add_CXXFLAGS) $(CXXFLAGS)" \
```

Comment:
Given that you've made this change to swap the order of `openfst_add_CXXFLAGS` and `CXXFLAGS`, can you change -O3 back to -O2 in `openfst_add_CXXFLAGS`? Right now, a semi-large change is adding different, unrelated changes. If you really want -O3 for your use case, you can do `CXXFLAGS="-O3" make`

Reply:
Good point! Done.